quickstart/exampledata/config/pipeline.yml

```diff
@@ -3,47 +3,51 @@ timeout: 0.1
 
 pipeline:
   - labelername:
-        type: labeler
-        schema: quickstart/exampledata/rules/labeler/schema.json
-        include_parent_labels: on
-        rules:
-            - quickstart/exampledata/rules/labeler/rules
+      type: labeler
+      schema: quickstart/exampledata/rules/labeler/schema.json
+      include_parent_labels: on
+      specific_rules:
+        - quickstart/exampledata/rules/labeler/rules
+      generic_rules:
+        - quickstart/exampledata/rules/labeler/rules
 
   - normalizer:
-     type: normalizer
-     specific_rules:
-       - quickstart/exampledata/rules/normalizer/specific/
-     generic_rules:
-       - quickstart/exampledata/rules/normalizer/generic/
-     regex_mapping: quickstart/exampledata/rules/normalizer/normalizer_regex_mapping.yml
+      type: normalizer
+      specific_rules:
+        - quickstart/exampledata/rules/normalizer/specific/
+      generic_rules:
+        - quickstart/exampledata/rules/normalizer/generic/
+      regex_mapping: quickstart/exampledata/rules/normalizer/normalizer_regex_mapping.yml
 
   - dropper:
-        type: dropper
-        rules:
-            - quickstart/exampledata/rules/dropper/rules
+      type: dropper
+      rules:
+        - quickstart/exampledata/rules/dropper/rules
 
   - pre_detector:
-     type: pre_detector
-     rules:
-       - quickstart/exampledata/rules/pre_detector/rules
-     pre_detector_topic: sre
-     tree_config: quickstart/exampledata/rules/pre_detector/tree_config.json
-     alert_ip_list: quickstart/exampledata/rules/pre_detector/alert_ips.yml
+      type: pre_detector
+      specific_rules:
+        - quickstart/exampledata/rules/pre_detector/rules
+      generic_rules:
+        - quickstart/exampledata/rules/pre_detector/rules
```

Comment:
I think that in the predetector, passing identical sets of rules as generic and specific will lead to duplicate detection results from separate trees. This will not cause duplicate events in the pre_det-topic, but it might generate duplicate elements in the list of detection results per processed event.